go/scripts/start-tokenizerbackend-dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-sudo rm -f ./outputs/aws_resources.yaml
+sudo rm -f ./outputs/aws_resources.json
```

Comment:
I think we want to move this folder wipe into the CLI itself.  If someone uses our script here then it will work, but if they call lunasec directly in their project, like we tell them to, more than one time, it will get confused,  the config file ends up as a folder somehow, and you get a cryptic error.  

I also think we want to change the name of this folder so that it makes sense in any project other than our own.  `lunasec-deployment-info` or something..